microceph/ceph/operations.go

```diff
@@ -128,17 +124,29 @@ func (o *CheckNonOsdSvcEnoughOps) Run(name string) error {
 		"mgr": 0,
 		"mds": 0,
 	}
+	totals := map[string]int{
+		"mon": 0,
+		"mgr": 0,
+		"mds": 0,
+	}
 	for _, service := range services {
 		// do not count the service on this node
 		if service.Location != name {
 			remains[service.Service]++
 		}
+		totals[service.Service]++
```

Comment:
ok, this line is a little confusing to me. On L134-L136 we conditionally increment the service count, and on L137 we do it unconditionally ? Can you please a code comment which explains this behavior ?

Reply:
Sure, I added comments and refactored the code too for clarity. :)

Reply:
(force pushed to add the sign off trailer to the new commits)

Reply:
much better, thanks :D